server/handlers/user_handler.go

```diff
@@ -80,6 +80,42 @@ func (h *Handler) GetUsers(w http.ResponseWriter, req *http.Request, _ *models.P
 	fmt.Fprint(w, string(resp))
 }
 
+// swagger:route GET /api/identity/users/keys UserKeysAPI idGetAllUsersKeysHandler
```

Comment:
Run 
```
make swagger-build
```

To update the swagger files with these new docs

Reply:
Output:

```
 make swagger-docs-build
swagger generate spec -o ./docs/_data/swagger.yml --scan-models; \
	swagger flatten ./docs/_data/swagger.yml -o ./docs/_data/swagger.yml --with-expand --format=yaml
object has no key "keys"
make: *** [swagger-docs-build] Error 1
```